ui/Makefile

```
@@ -1,27 +1,31 @@
+	# field in package.json, so the version is pinned in exactly one place.
+	# The sudo fallback covers images where corepack's shim dir is root-owned.
+	corepack enable pnpm 2>/dev/null || sudo corepack enable pnpm
+	pnpm install
 
 build: setup
-	yarn build
+	pnpm build
 
 release: build
 
 clean:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)